epsagon/wrappers/python_function.py

```diff
@@ -52,8 +54,7 @@ def wrap_python_function(func, args, kwargs):
         raise
     finally:
         try:
-            if not epsagon.trace.trace_factory.get_trace().metadata_only:
-                runner.resource['metadata']['return_value'] = result
+            runner.add_json_field('python.function.return_value', result)
```

Comment:
Does changing the way the return value is sent (the name of the tag) requires adaptations in the backend as well?

Reply:
Nope